assembl/static2/css/components/announcement.scss

```diff
@@ -1,25 +1,32 @@
 .announcement {
   text-align: center;
+  margin-left: 4.5%;
+  margin-right: 4.5%;
+  padding-right: 20%;
+  padding-bottom: $margin-section-xxs;
 
   .announcement-title {
     .announcement-title-text {
-      margin-top: 0.5em;
+      margin-top: 11px;
     }
     text-align: left;
   }
 
   .announcement-statistics {
     display: inline-block;
     text-align: center;
-    margin-top: 1.5em;
+    margin-top: 37.33px;
```

Comment:
37.33px??

Reply:
pixel decimals are supported, it was the value in the invision css

Reply:
For your information Norman, Roxana's mock up are not pixel perfect so you can use the inVision inspector to have an idea but you don't need to use exactly the same value.